torch/distributed/distributed_c10d.py

```diff
@@ -428,6 +481,11 @@ def pg_to_tag(self) -> Dict[ProcessGroup, str]:
         global _pg_to_tag
         return _pg_to_tag
 
+    @property
+    def pg_coalesce_state(self) -> Dict[ProcessGroup, List[Union[CollOp, P2POp]]]:
+        global _pg_coalesce_state
```

Comment:
(this is before this PR) Curious, why do we need `global` here? I assume we only need to read the `_pg_coalesce_state` py obj in this function?

Reply:
It's superfluous, a matter of style.

Reply:
Moved inside `_World` now. Thanks for the insight.